### DISCUSSION TITLE
Accounts for GoCardless not shown

### DISCUSSION THREAD

**Discussion by mamob07** [firefly-iii/firefly-iii#9507] [Firefly III] [answered]
why doesn't it show me the accounts here?

![{877F2B1A-9803-41CA-B1F3-3F0BF83BEF8B}](https://github.com/user-attachments/assets/de653de8-ffc8-4320-bc5c-c3be2ae8a2e1)

**Comment by JC5** [comment]
I don't know. I would need a bit more details to answer your question.

**Reply by mamob07** [reply, marked as answer]
Hello, I fixed deleting the existing bank account in storage files and then I added it back.
